src/GLPIKey.php

```diff
@@ -126,25 +127,12 @@ public function keyExists()
     }
 
     /**
-     * Get GLPI security key used for decryptable passwords
-     *
-     * @return string|null
+     * Get GLPI security key used for decryptable contents
      */
-    public function get(): ?string
+    public function get(): string
     {
-        if (!file_exists($this->keyfile)) {
-            trigger_error('You must create a security key, see security:change_key command.', E_USER_WARNING);
-            return null;
-        }
-        if (!is_readable($this->keyfile) || ($key = file_get_contents($this->keyfile)) === false) { //@phpstan-ignore theCodingMachineSafe.function
-            trigger_error('Unable to get security key file contents.', E_USER_WARNING);
-            return null;
-        }
-        if (strlen($key) !== SODIUM_CRYPTO_AEAD_XCHACHA20POLY1305_IETF_KEYBYTES) {
-            trigger_error('Invalid security key file contents.', E_USER_WARNING);
-            return null;
-        }
-        return $key;
+        $this->ensureKeyIsValid();
+        return file_get_contents($this->keyfile);
```

Comment:
This makes it read the key file twice every time it needs to perform an encrypt/decrypt. Should be avoided.

Reply:
That's the downside of having 2 methods. The footprint of reading this local file et probably close to null. So I choose this approach (2 methods vs 1), it looks really better than having a single method that does both (checking key & show the message)  and violate SRP principle.
Maybe we can have a local variable to keep the errors and avoid 2 calls to filegetcontents.